### PR TITLE
For 150: updated list of implemented metrics

### PR DESCRIPTION
**Pull request by paulodamaso**
For !50:
- updated the list of implemented metrics

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#433](https://github.com/yegor256/jpeek/issues/433) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/f6b50020-8e43-4429-b38a-c92c8dfb7655); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @paulodamaso[/z](https://www.0crat.com/u/paulodamaso) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: README.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ _Revisiting Class Cohesion: An empirical investigation on several systems_,<br/>
 Journal of Object Technology, vol.7, no.6, 2008,
 [PDF](http://www.jot.fm/issues/issue_2008_07/article1.pdf).
 
-Here is a list of metrics we already implement:
+###Here is a list of metrics we already implement:
```
Comment (Vatavuk) [inline README.md]: @paulodamaso I believe grammar is incorrect, `Here is a list of metrics that we've already implemented`

**Comment by victornoel** [comment]
@Vatavuk @paulodamaso I didn't see you took care of this, but I got the task assigned earlier and tackled it, see #435. I believe it's more complete that this one.

**Comment by Vatavuk** [comment]
@paulodamaso ping

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
Pull request [#433](https://github.com/yegor256/jpeek/issues/433) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues during next CT

@paulodamaso please make sure all issues found by code reviewers would be fixed by you if you were pull-request author again

I've seen that this is no longer actual, but policy is policy

**Comment by paulodamaso** [comment]
@sereshqua Will do, thanks

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request closed without merging**